satellite/metainfo/metainfo.go

```diff
@@ -548,7 +551,23 @@ func (endpoint *Endpoint) ValueAttributeInfo(ctx context.Context, req *pb.ValueA
 		return &pb.ValueAttributionResponse{}, err
 	}
 
-	// TODO: add valueattribution DB access functions added in new PR.
+	// check if already bucket exists
+	_, err = endpoint.partnerinfo.Get(ctx, req.GetBucketName())
+	if err != nil {
+		// no rows
+		if valueattribution.ErrBucketNameNotFound.Has(err) {
+			_, err = endpoint.partnerinfo.Insert(ctx, &valueattribution.PartnerInfo{
+				PartnerID:  req.GetPartnerId(),
+				BucketName: req.GetBucketName(),
+			})
+			if err != nil {
+				endpoint.log.Sugar().Info("related partner info could not be attributed \n")
```

Comment:
does the logger need a newline at the end?

Reply:
refactored the code and removed the logs are they are redundant